.github/workflows/swift-toolchain.yml

```diff
@@ -986,9 +1268,9 @@ jobs:
                 -D CURL_USE_LIBSSH2=NO `
                 -D CURL_USE_MBEDTLS=NO `
                 -D CURL_USE_OPENSSL=NO `
-                -D CURL_USE_SCHANNEL=YES `
+                -D CURL_USE_SCHANNEL=${{ matrix.windows_cmake_flag }} `
                 -D CURL_USE_WOLFSSL=NO `
-                -D CURL_WINDOWS_SSPI=YES `
+                -D CURL_WINDOWS_SSPI=${{ matrix.windows_cmake_flag }} `
```

Comment:
We could use a ternary here instead - `matrix.os == 'Windows' && 'YES' || 'NO'`.